indexing-handlers/src/main/java/no/unit/nva/indexing/handlers/IndexCreationHandler.java

```diff
@@ -0,0 +1,35 @@
+package no.unit.nva.indexing.handlers;
+
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import no.unit.nva.search.IndexingClient;
+import nva.commons.core.JacocoGenerated;
+
+public class IndexCreationHandler implements RequestHandler<Object, String> {
+
+    public static final String RESOURCES = "resources";
+    public static final String DOIREQUESTS = "doirequests";
+    public static final String MESSAGES = "messages";
+    public static final String SUCCESS_RESPONSE = "OK";
+    private final IndexingClient indexingClient;
+
+    @JacocoGenerated
+    public IndexCreationHandler() {
+        this(new IndexingClient());
+    }
+
+    public IndexCreationHandler(IndexingClient indexingClient) {
+        this.indexingClient = indexingClient;
+    }
+
+    @Override
+    public String handleRequest(Object input, Context context) {
+        var indicesClientWrapper = indexingClient.getIndicesClientWrapper();
```

Comment:
why do you need to expose the indicesClientWrapper and then send it back to the same obejct as parameter? There is no need for the indicesClientWrapper to be exposed.

Reply:
Thought of using the same indices client wrapper class instead of creating it each time when creating index, but as you said instead of exposing it to the client better to use it in indexingclient itself will update the code